aretomo/protocols/protocol_aretomo.py

```diff
@@ -357,7 +362,7 @@ def convertInputStep(self, tsId: str, tsFn: str):
 
         # Generate angle file
         angleFilePath = self.getFilePath(tsFn, tmpPrefix, ".tlt")
-        self._generateTltFile(ts, angleFilePath)
+        ts.generateTltFile(angleFilePath)
```

Comment:
this does not generate a proper tlt file for aretomo. See the removed function below

Reply:
Hi, @azazellochg. With the new and updated methods in the TiltSeries class, it seems to work fine, all the tests with and without excluded views pass. The main problem I see in Aretomo2-1.1.3 is that the reconstruction is always looking for a .Aln file, so we'll have to generate it in the convert when only the tomograms reconstruction is requested. This is what I get when trying to do that:

The command:

`/home/jjimenez/scipion3/software/em/aretomo2-1.1.3/bin/AreTomo2_1.1.3_Cuda121 -InMrc Runs/000123_ProtAreTomoAlignRecon/tmp/TS_03/03.mrc -OutMrc Runs/000123_ProtAreTomoAlignRecon/extra/TS_03/03.mrc -OutImod 0 -AngFile Runs/000123_ProtAreTomoAlignRecon/tmp/TS_03/03.tlt -VolZ 1200 -OutBin 4 -FlipInt 0 -FlipVol 1 -PixSize 1.35 -Kv 300.0 -DarkTol 0.7 -AmpContrast 0.07 -Gpu 0 -Cs 2.7 -Align 0 -TiltAxis 0.0 0 -TiltCor 0 -AlignZ 800 -Sart 15 5`

And the error:

Error: tilt angles cannot be found.

But it does exist.

BUT, if I pass the aln file and not the tlt file (as it was before), then it reconstructs the tomogram. This is the corresponding command:

`/home/jjimenez/scipion3/software/em/aretomo2-1.1.3/bin/AreTomo2_1.1.3_Cuda121 -InMrc Runs/000223_ProtAreTomoAlignRecon/extra/TS_03/03.mrc -OutMrc Runs/000223_ProtAreTomoAlignRecon/extra/TS_03/tomo_03.mrc -OutImod 0 -VolZ 1200 -OutBin 4 -FlipInt 0 -FlipVol 1 -PixSize 1.35 -Kv 300.0 -DarkTol 0.7 -AmpContrast 0.07 -Gpu 0 -Cs 2.7 -Align 0 -TiltAxis 0.0 0 -TiltCor 0 -AlignZ 800 -Sart 15 5 -AlnFile Runs/000223_ProtAreTomoAlignRecon/extra/TS_03/03.aln`

I can take care of the aln file generation if it's ok for you.

Reply:
Oh! I'll add the dose column. Thanks, @azazellochg

Reply:
Hi, @azazellochg, I've add this but in the new method from the TiltSeries model, so it goes together with the exclusion of views. See here --> https://github.com/scipion-em/scipion-em-tomo/blob/jj_fix_ts_clone/tomo/objects.py#L566